src/Artsy/Router/Components/Boot.tsx

```diff
@@ -28,7 +29,16 @@ export interface BootProps {
   headTags?: JSX.Element[]
 }
 
-const { GlobalStyles } = injectGlobalStyles(`
+// Build global responsive styles
+const reactResponsiveStyles = createMediaStyle()
+
+// FIXME: When we update to latest @types/styled-components `suppressMultiMountWarning`
+// issue will be fixed
+const { GlobalStyles } = injectGlobalStyles<{
+  suppressMultiMountWarning: boolean
```

Comment:
Once we're on the 4.x branch i'll follow up and remove some of these unneeded props.